### PR TITLE
feat(cli): shows uninstalling status when a package is being uninstalled using the list and describe command (#654)

### LINKED ISSUE #654: Show uninstalling state in `list` and `describe` command
**Is your feature request related to a problem? Please describe.**
As a follow up of #456 we should also show "uninstalling" in the CLI, i.e. `list` and `describe` commands, in order to keep CLI and UI consistent. 

**Describe the solution you'd like**
As defined in #456, when a package is in the progress of being uninstalled, its status should be "Uninstalling" instead of "Pending". 

**Describe alternatives you've considered**

**Additional context**
#456

### PR DESCRIPTION
**Pull request by bishalbera**
Closes #654

## 📑 Description

Shows uninstalling state when a package is being uninstalled using the `list` and `describe` command.

Screenshot 2024-05-23 at 3 32 21 PM

Screenshot 2024-05-23 at 3 33 37 PM

## ✅ Checks
- [ ] My code requires changes to the documentation
- [ ] I have updated the documentation as required

### CODE REVIEW

**Comment by christophenne** [comment]
Hi @bishalbera, thanks for your PR – since there are some conflicts now, could you please rebase your branch?

**Comment by bishalbera** [comment]
Hey @christophenne so this is my first time using the rebase command 😅 and I've solved those conflicts but after rebasing how do I commit those changes to main branch. It's showing that my branch is clean nothing to commit :( . Can you please guide me ?
Screenshot 2024-05-27 at 4 07 42 PM

**Comment by christophenne** [comment]
Looks like everything is commited already, I think. You only need to force-push to your branch, and the updated commit should appear here.

**Comment by bishalbera** [comment]
Yup it worked but but the checks were not successull because of the commit message . Really sorry for that . @christophenne

**Comment by christophenne** [comment]
> Yup it worked but the checks were not successfull because of the commit message . Really sorry for that . @christophenne

Ok no worries, I need to test it anyway. Until then you can try to change the commit message, see e.g. https://stackoverflow.com/a/179147 – you will need to force push again afterwards.

**Comment by bishalbera** [comment]
@christophenne Thanks, it's a great share. Just changed the commit message

**Comment by christophenne** [comment]
> @christophenne Thanks, it's a great share. Just changed the commit message

@bishalbera I don't think changing the commit message worked, it' still too long.

**Comment by bishalbera** [comment]
ok let me try again :( @christophenne .

**Review by christophenne — APPROVED** [review]
